validator/src/commands/exit/mod.rs

```diff
@@ -102,16 +104,79 @@ pub fn execute(matches: &ArgMatches, ledger_path: &Path) -> Result<()> {
     }
 
     let admin_client = admin_rpc_service::connect(ledger_path);
-    admin_rpc_service::runtime().block_on(async move { admin_client.await?.exit().await })?;
+    let validator_pid =
+        admin_rpc_service::runtime().block_on(async move { admin_client.await?.exit().await })?;
```

Comment:
Note from testing - if new (PID returned here) `agave-validator` calls exit on old `agave-validator`, this command will error out with:
```
Validator command failed: admin rpc error: Failed to parse server response as u32: invalid type: null, expected u32
```
The exit has still been initiated, we just can't monitor. Can think about whether we want to do anything more or not